config/routes.rb

```diff
@@ -16,7 +16,9 @@
   resources :comments, :only => [:destroy, :edit, :update]
   resources :users, :only => [:show, :edit, :update]
   namespace :admin do
-    resources :users
+    resources :users do
+      resources :badges, :only => [:create, :destroy]
```

Comment:
actually you should generate this controller:

```
rails g controllers admin/badges
```

Since it's nested under the `admin` namespace.